docs/root/version_history/current.rst

```diff
@@ -1,14 +1,47 @@
 1.15.0 (Pending)
 ================
 
-Changes
+
+Incompatible Behavior Changes (things that expected to cause an incompatibility if they apply; changes required)
```

Comment:
```suggestion
Incompatible Behavior Changes (changes that are expected to cause an incompatibility if applicable; deployment changes are likely required)
```

Reply:
style preference: I would probably put the description in italics under the header, same elsewhere.